docs/source/testing.rst

```diff
@@ -92,46 +102,80 @@ settings.local.yaml:
+
+Debugging Options
+=================
+
+Running PDB in dockerized koku
+------------------------------
+
 To use pdb while running the koku-server in docker:
-Ensure all migrations are run.
-Stop the server `docker-compose stop koku-server`
-Run the server with service-ports: `docker-compose run —service-ports koku-server`
-Breakpoints will now be stopped in this terminal window.
-
-To test a specific file using tox, edit this line, for example:
-`coverage run {toxinidir}/koku/manage.py test --noinput -v 2 {posargs: masu.test.database}`
-This will selectively run only the masu database tests, instead of running all of the rest of the tox tests.
-
-If you observe the following error in the tox tests, you may sometimes ignore it, due to tox not setting DEBUG=TRUE, to fix this you can export the variable to be true.
-```
-======================================================================
-FAIL: test_delete_single_provider_skips_delete_archived_data_if_customer_is_none (api.provider.test.tests_models.ProviderModelTest)
-Assert the delete_archived_data task is not called if Customer is None.
-----------------------------------------------------------------------
-Traceback (most recent call last):
-  File "/usr/local/Cellar/python/3.7.4_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/mock.py", line 1209, in patched
-    return func(*args, **keywargs)
-  File "/Users/nbonilla/Documents/Koku/koku/koku/api/provider/test/tests_models.py", line 43, in test_delete_single_provider_skips_delete_archived_data_if_customer_is_none
-    self.aws_provider.delete()
-AssertionError: no logs of level WARNING or higher triggered on api.provider.models
-
-----------------------------------------------------------------------
-Ran 756 tests in 2025.258s
-FAILED (failures=1)
-```
-
-# Gotchas:
-- The logger is disabled by default during unit tests. If you are building a unit test that asserts that a log occurs, you must re-enable the logger. For example:
-
-```
-import logging
-with self.assertLogs(logger='masu.external.downloader.aws.aws_report_downloader', level='WARN') as cm:
-            logging.disable(logging.NOTSET)
-            self.aws_report_downloader._remove_manifest_file("None")
-            self.assertEqual(['WARN: Could not delete manifest file at'], cm.output)
-```
+    1. Ensure all migrations are run.
+    2. Stop the server `docker-compose stop koku-server`
+    3. Run the server with service-ports: :code:`docker-compose run —service-ports koku- 
+       server`
+    4. set a breakpoint using :code:`import pdb; pdb.set_trace()`
+
+Breakpoints should now be stopped at, in this terminal window.
+
+
+Running PDB in IQE
+------------------
+To run PDB while using dockerized IQE: 
+    Start a shell session in the docker container that runs IQE::
+
+        koku/testing/run_test.sh bash
+
+To run a specific test, use the following command. Replace :code:`-k` flag's input to a specific test you want to run::
```

Comment:
Similar to a previous comment above. This directive is complex and indirect. It should be presented in a more direct way. 

Saying "replace X with Y" has an implicit assumption that the reader knows more about the test suite than they may actually know. It's hard to approach this kind of a sentence as a newbie who may not know what to replace the example with. 

```suggestion
To run a specific subset of the integration test suite (IQE), you can specify a group of tests or a single test using the :code:`-k` flag. You can locate a list of test components <describe where to find them>. Here is one example::
```